management/management.go

```diff
@@ -168,6 +171,7 @@ func New(domain string, options ...Option) (*Management, error) {
 	m.ClientGrant = (*ClientGrantManager)(&m.common)
 	m.Connection = (*ConnectionManager)(&m.common)
 	m.CustomDomain = (*CustomDomainManager)(&m.common)
+	m.CustomPrompt = (*CustomPromptManager)(&m.common)
```

Comment:
Please correct me if I am wrong but the endpoints in question seem to be under the Prompts resource within the API Docs Structure -> https://auth0.com/docs/api/management/v2/prompts/get-partials

Having these now on a separate manager might be confusing, when I'd expect to find them under the Prompt manager as it's something that should be that resource's responsibility. 

Also within the API docs I only see GET and PUT but no DELETE or POST.